systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingST.java

```diff
@@ -0,0 +1,785 @@
+import io.strimzi.systemtest.utils.StUtils;
+import io.strimzi.test.TestUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Random;
+import java.util.Stack;
+import java.util.stream.Collectors;
+
+import static io.restassured.RestAssured.given;
+
+import static io.strimzi.systemtest.Constants.REGRESSION;
+import static io.strimzi.test.TestUtils.getFileAsString;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anything;
+import static org.hamcrest.Matchers.greaterThan;
+
+@Tag(REGRESSION)
+public class TracingST extends AbstractST {
+
+    private static final String NAMESPACE = "tracing-cluster-test";
+    private static final Logger LOGGER = LogManager.getLogger(TracingST.class);
+    private static final String JO_INSTALL_DIR = "../systemtest/src/test/resources/tracing/jaeger-operator/";
+    private static final String JI_INSTALL_DIR = "../systemtest/src/test/resources/tracing/jaeger-instance/";
+
+    private static final String JAEGER_PRODUCER_SERVICE = "hello-world-producer";
+    private static final String JAEGER_CONSUMER_SERVICE = "hello-world-consumer";
+    private static final String JAEGER_KAFKA_STREAMS_SERVICE = "hello-world-streams";
+    private static final String JAEGER_MIRROR_MAKER_SERVICE = "my-mirror-maker";
+    private static final String JAEGER_KAFKA_CONNECT_SERVICE = "my-target-connect";
+
+    private static final String TOPIC_NAME = "my-topic";
+    private static final String TOPIC_TARGET_NAME = "cipot-ym";
+
+    private Stack<String> jaegerConfigs = new Stack<>();
+
+    @Test
+    void testJaegerService() {
+
+        given()
+            .when()
+                .relaxedHTTPSValidation()
+                .contentType("application/json").get("/jaeger/api/services")
+            .then()
+                .statusCode(200)
+                .body(anything())
+            .log().all();
+    }
+
+    @Test
+    void testProducerService() {
+        Map<String, Object> configOfSourceKafka = new HashMap<>();
+        configOfSourceKafka.put("offsets.topic.replication.factor", "1");
+        configOfSourceKafka.put("transaction.state.log.replication.factor", "1");
+        configOfSourceKafka.put("transaction.state.log.min.isr", "1");
+
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 1, 1)
+                .editSpec()
+                    .editKafka()
+                        .withConfig(configOfSourceKafka)
+                        .withNewPersistentClaimStorage()
+                            .withNewSize("100")
```

Comment:
```suggestion
                            .withNewSize("10")
```
On some of our environments we have PVs only with 10Gb available